ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -39,8 +39,13 @@ QtObject {
     property var overview: walletSectionOverview
     property bool balanceLoading: overview.balanceLoading
     property var accounts: walletSectionAccounts.accounts
-    property var receiveAccounts: walletSectionSend.accounts
-    property var selectedReceiveAccount: walletSectionSend.selectedReceiveAccount
+    property var selectedReceiveAccount: {
```

Comment:
Store is not good place for that because it's more that just simple exposition of backend's props/methods. There is specialized component `ModelEntry` that allows to track single item from the model by key easily (but should be instantiated outside of the store).